source/_components/alarm_control_panel.lupusec.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Lupusec Alarm Control Panel"
+description: "Instructions on how to setup the Lupusec Alarm control panel within Home Assistant."
+date: 2018-10-03 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lupusec.jpg
+ha_category: Alarm
+ha_release: 0.79
```

Comment:
`0.79` has been shipped already some time ago. The next possible release would be `0.81`.